services/src/main/java/io/grpc/services/HealthCheckingLoadBalancerFactory.java

```diff
@@ -421,6 +425,9 @@ void handleResponse(HealthCheckResponse response) {
       void handleStreamClosed(Status status) {
         if (Objects.equal(status.getCode(), Code.UNIMPLEMENTED)) {
           disabled = true;
+          logger.log(
+              Level.SEVERE, "Health-check with {0} is disabled: {1}",
```

Comment:
Make it clear the status came from the server. For example, "Health-check with {0} is disabled. Server returned: {1}"

Reply:
Done.